web/styles/left_sidebar.css

```diff
@@ -288,7 +288,12 @@ li.show-more-topics {
         & li#show-more-direct-messages {
             cursor: pointer;
             padding-right: 26px;
-            padding-left: 6px;
+            /* TODO: Rewrite the show-more and show-less
+               buttons as grids alongside the left sidebar
+               header rewrites. This padding value
+               is a stopgap to provide the same lefthand
+               alignment as the "back to streams" text. */
+            padding-left: 10px;
```

Comment:
"more conversations" feels too far left to me -- it looks like the text is left of the "DIRECT MESSAGES" and "STREAMS" headings, and that feels wrong.

Reply:
Wholly agreed--and yet, it's where it is because it aligns with the "back to streams" row in the zoomed-in view. This is right now on `main`/CZO:

<img width="281" alt="lsb-on-main" src="https://github.com/zulip/zulip/assets/170719/2adbb5b7-aeaf-4f1f-aacf-8a450bce4495">

My plan is to open a design discussion about that when I hit those and other similar rows, probably in tandem with the heading rows. @terpimost has [a very different layout and typographic treatment](https://terpimost.github.io/zulip-sidebar/) for those, which I think we can get placed well ahead of 8.0.

Reply:
Sure thing. Just pushed a fresh commit with alignment for these two things.

"back to streams" now aligns with the DIRECT MESSAGES header:
<img width="305" alt="Screenshot 2023-11-01 at 10 03 45 AM" src="https://github.com/zulip/zulip/assets/170719/c90151a5-bd4a-4936-99d8-7c20180d0fd1">

"more conversations" takes its cue from "more topics," which aligns with the topic above; "more conversations" now aligns with the DM name above:
<img width="305" alt="Screenshot 2023-11-01 at 10 03 33 AM" src="https://github.com/zulip/zulip/assets/170719/f80b4304-e6d5-46c5-84d1-d678d65df640">